staging/src/k8s.io/apiserver/Godeps/Godeps.json

```diff
@@ -354,6 +354,14 @@
 			"ImportPath": "github.com/dgrijalva/jwt-go",
 			"Rev": "01aeca54ebda6e0fbfafd0a524d234159c05ec20"
 		},
+		{
+			"ImportPath": "github.com/docker/distribution/digestset",
```

Comment:
This looks spurious

Reply:
Sorry for that, I pushed from the wrong machine :) should have been fixed now.